src/Types/EnergyAmount.cs

```diff
@@ -16,4 +17,41 @@ namespace Concordium.Sdk.Types;
     public byte[] ToBytes() => Serialization.ToBytes(this.Value);
 
     internal static EnergyAmount From(Grpc.V2.Energy energy) => new(energy.Value);
+
+    /// <summary>
+    /// Add Energy amounts.
+    /// </summary>
+    /// <exception cref="ArgumentException">The result odoes not fit in <see cref="ulong"/></exception>
+    public static EnergyAmount operator +(EnergyAmount a, EnergyAmount b)
```

Comment:
**Discussion:** I am not so sure about having this implementation. Since this makes it checked by default, which is different from `+` for other types.
Likewise for the `-` below.

Reply:
It's basically copied from `CcdAmount` so that has the same issue. I don't have much opinion on it, what about you? Did you want to remove it or what?

Reply:
okay, lets just stick with it for now then